src/tests/api/ApiTestSteps.ts

```diff
@@ -59,7 +59,7 @@ export async function startStubServiceAndReturnSessionId(stubPayload: StubStartR
 
 export async function stubStartPost(stubPayload: StubStartRequest): Promise<AxiosResponse<StubStartResponse>> {
 	const path = constants.DEV_IPV_F2F_STUB_URL;
-	if (constants.THIRD_PARTY_CLIENT_ID) {
+	if (constants.THIRD_PARTY_CLIENT_ID === "SandboxJourneyFlow") {
```

Comment:
Looking at this again I think the previous statement was correct as its sets the value in the stub payload. 

Do we know why we added this line
		delete stubPayload.yotiMockID;

Im assuming its because yoti set this in reality.
In which case this should probably be

	if (constants.THIRD_PARTY_CLIENT_ID) {
		stubPayload.clientId = constants.THIRD_PARTY_CLIENT_ID;
	if (constants.THIRD_PARTY_CLIENT_ID === "SandboxJourneyFlow") {
		delete stubPayload.yotiMockID;
}
}